pkgs/development/compilers/llvm/16/compiler-rt/freebsd-i386.patch

```diff
@@ -0,0 +1,21 @@
+--- a/lib/builtins/fp_lib.h	1969-12-31 16:00:01.000000000 -0800
++++ b/lib/builtins/fp_lib.h	2023-12-21 23:39:36.066927293 -0800
+@@ -26,18 +26,6 @@
+ #include <stdbool.h>
+ #include <stdint.h>
+ 
+-// x86_64 FreeBSD prior v9.3 define fixed-width types incorrectly in
```

Comment:
So now it's defined correctly?

Reply:
Indeed, and we don't support FreeBSD<14, maybe in the future to be <13. I suspect the problematic part of this patch was just including sys/param.h, but regardless it prevented literally any i386 freebsd code (in this case, the bootloader) from compiling.